lib/renderer.js

```diff
@@ -1164,10 +1183,12 @@
         // 4. The "include-ancestor-masks" config option is NOT set
         // 5. None of the layers has zero bounds. Sometimes they aren't computed and set to all 0's
         // 6. The layer is clipped, in which case layer.bounds is the clipped size and not the layer size
-        var hasComplexTransform = (component.hasOwnProperty("scale") && component.scale % 1 !== 0) ||
-                component.hasOwnProperty("width") || component.hasOwnProperty("height"),
+        var layer = component.layer,
+            hasComplexTransform = layer &&
```

Comment:
I'm not clear when you can't scale in this case. The main reason for this claus is for the case where it's possible the cached bounds information for getDocumentInfo is not correct. The comps should have the same bounds as document.bounds which should always be correct. This is just, should be pre-generate the image at 100% to know it's bounds for sure or can safely calculate it off the information we already have.